lib/Http/JsonResponse.php

```diff
@@ -109,11 +116,18 @@ public static function error(string $message,
 	}
 
 	/**
-	 * @param mixed[] $data
+	 * @param Throwable $error
+	 * @param int $status
+	 * @param array|JsonSerializable|bool|string $data
+	 *
+	 * @return static
 	 */
 	public static function errorFromThrowable(Throwable $error,
 		int $status = Http::STATUS_INTERNAL_SERVER_ERROR,
-		array $data = []): self {
+		$data = []): self {
+		if (!is_array($data)) {
```

Comment:
can we not keep the unwrapped value?

Reply:
Hm, not really since we merge the data and serialized exception which is `array|null`

Reply:
You are right